packages/clerk-js/src/core/clerk.ts

```diff
@@ -969,7 +969,7 @@ export class Clerk implements ClerkInterface {
 
   public buildAfterMultiSessionSingleSignOutUrl(): string {
     if (!this.#options.afterMultiSessionSingleSignOutUrl) {
-      return this.buildAfterSignOutUrl();
+      return this.environment ? `${this.environment.displayConfig.signInUrl}/choose` : this.buildAfterSignOutUrl();
```

Comment:
❓ if the signInUrl is defined shouldn't that be used instead of the values retrieved the environment ?